stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -26,13 +28,20 @@
     "key": "x_com_ibm_ariel.identity_host_name",
     "cybox": false
   },
-  "qidname": {
-    "key": "x_com_ibm_ariel.qid_name",
-    "cybox": false
-  },
+  "qidname": [
+    {
+      "key": "x_com_ibm_ariel.qid_name",
```

Comment:
we should set this field as `event_name` instead of `qid_name`. 
In that case we don't even need to add that in `"x_risk_finding.event_name"`